euphonic/development/02_releases.md

```diff
@@ -58,66 +39,56 @@ accurate. e.g. if you are releasing v0.5.2, and you see a deprecated directive t
-which will build and upload a source and various wheels to PyPI. It
-should also start a workflow `create-landing-page.yml` which should
-publish a new version page to the `gh-pages` branch
-
-## 9. Test PyPI releases
-There is a `test_release.yml` workflow that must be triggered manually with
-a workflow dispatch. Run the workflow with the release version to check all
-the PyPI builds succeed. The Conda builds may fail if a Conda release hasn't
-been made yet
-
-## 10. Test versioned landing page
-The above mentioned  `create-landing-page.yml` workflow should have created
-a versioned page e.g. https://pace-neutrons.github.io/Euphonic/versions/v0.6.2.html,
-check it looks sensible
-
-## 11. Update conda-forge package
-Wait for the conda-forge bot to open a PR in https://github.com/conda-forge/euphonic-feedstock,
-this may take a few hours. Make sure all the tests pass, and merge the PR.
-
-## 12. Test conda-forge package
+## 2. Create a release branch and wait for tests to pass.
+
+Create a branch with "release" somewhere in the name and push to
+GitHub. This will trigger a "run-tests" workflow which can be
+monitored here[https://github.com/pace-neutrons/Euphonic/actions].
+
+> **NOTE:** unlike the automatic PR tests, this will include all
+> supported platforms and python versions.
+> A full test can also be run on a PR by using a manual dispatch
+> of run-tests, targeting the appropriate branch.
+
+## 3. Run the Release workflow
+
+In the [Github Actions](https://github.com/pace-neutrons/Euphonic/actions)
+sidebar choose "Create a release" (release.yml) workflow and run
+specifying the release branch and proposed version number.
+
+- Pre-releases should have text after the semver number, such as "v1.4.1rc2".
+- It should generally be safe to enable "Make release and push to PyPI" without a dry-run: the workflow will exit early if a step fails.
+
+### Fixing a failed release workflow
+
+- Small problems can be fixed on the release branch. Consider making a pull-request to master so these can be reviewed more easily; the branch can still be used to release.
+- If something went wrong after the "Bump version number" step, you will need to delete the new git tag from https://github.com/pace-neutrons/Euphonic/tags
+- If something went wrong after making a Github release, you will need to delete the release from https://github.com/pace-neutrons/Euphonic/releases
+- If good packages were already uploaded to PyPI but something went wrong producing the landing page, you don't need to redo the release. Fix it directly on the gh-pages branch.
+- When things are looking good, run the release workflow again.
+
+## 4. Check versioned landing page
+The release workflow should have called `create-landing-page.yml` to create
+a versioned page e.g. https://pace-neutrons.github.io/Euphonic/versions/v0.6.2.html;
+check it looks sensible.
+
+## 5. Post-release testing: PyPI
+There is a "Test PyPI release" (_test_release_release.yml_) workflow
```

Comment:
Consistency with earlier ones?
```suggestion
There is a "Test PyPI release" (`test_release_release.yml`) workflow
```

Reply:
Don't mind which "standard" is chosen (bare, italics, monospace, polka-dot papyrus font [may take some extra engineering to achieve]), just think we want some consistency.

Reply:
Yeah, it could be a bit more expressive! I usually prefer italics for filenames. But let's prioritise consistency and just backtick it all for now.